packages/protocol/script/DeployOnL1.s.sol

```diff
@@ -38,7 +38,7 @@ contract DeployOnL1 is Script {
 
     address public sharedSignalService = vm.envAddress("SHARED_SIGNAL_SERVICE");
 
-    address public treasury = vm.envAddress("TREASURY");
+    address public treasury = vm.envAddress("TREASURY"); // Q: which address will we use?
```

Comment:
You can use this value `0xdf09A0afD09a63fb04ab3573922437e1e637dE8b`, it's one of my hardware wallets.